libs/importer/src/importers/keeper/types/keeper-json-types.ts

```diff
@@ -1,41 +1,54 @@
-export interface KeeperJsonExport {
-  shared_folders?: SharedFoldersEntity[] | null;
-  records?: RecordsEntity[] | null;
-}
-
-export interface SharedFoldersEntity {
-  path: string;
-  manage_users: boolean;
-  manage_records: boolean;
-  can_edit: boolean;
-  can_share: boolean;
-  permissions?: PermissionsEntity[] | null;
-}
-
-export interface PermissionsEntity {
-  uid?: string | null;
-  manage_users: boolean;
-  manage_records: boolean;
-  name?: string | null;
-}
-
-export interface RecordsEntity {
-  title: string;
-  login: string;
-  password: string;
-  login_url: string;
+// See https://docs.keeper.io/en/user-guides/import-records-1/import-json#creating-a-custom-.json-file-for-import-into-keeper
+// for reference on the Keeper JSON format. It's not comprehensive but covers the main fields.
+
+export type KeeperJsonExport = {
+  records?: Record[] | null | undefined;
+  shared_folders?: SharedFolder[] | null | undefined;
```

Comment:
`?` already adds `undefined` to the type.

```suggestion
  records?: Record[] | null;
  shared_folders?: SharedFolder[] | null;
```